core-services/release-controller/_repos/ocp-4.14-rhel92.repo

```diff
@@ -36,7 +36,7 @@ failovermethod = priority
 
 [rhel-9.2-nfv]
 name = rhel-9.2-nfv
-baseurl = https://cdn.redhat.com/content/e4s/rhel9/9.2/x86_64/nfv/os/
```

Comment:
The naming has changed for RHEL 9 and we need to keep `e4s` here. `tus` was the old name for RHEL 8.

Reply:
Updated, thanks!